go.mod

```diff
@@ -5,6 +5,7 @@ go 1.12
 require (
 	github.com/gogo/protobuf v1.2.1 // indirect
 	github.com/golang/protobuf v1.3.1
+	github.com/golangci/golangci-lint v1.16.0 // indirect
```

Comment:
It looks like a go get github.com/golangci/golangci-lint was done with GO111MODULE=on in the root dir, it's better to revert this to master as it's an optional dependancy